src/Utilities/PaymentUtils.res

```diff
@@ -36,6 +40,15 @@ let paymentListLookupNew = (
     walletToBeDisplayedInTabs->Array.push("apple_pay")
   }
 
+  if (
+    !paymentMethodListValue.collect_billing_details_from_wallets &&
+    !areAllPaypalRequiredFieldsPreFilled &&
+    isPaypalRedirectFlow &&
```

Comment:
It seems we are not using `isShowPaypal` for deciding whether it should be added in tabs. Are we going to ignore if the merchant passes PayPal as `never` in the wallets @PritishBudhiraja?

Reply:
agreed. @sakksham7 let's add this check too. we forgot to add.

Reply:
Updated